.github/workflows/windows.yml

```diff
@@ -0,0 +1,21 @@
+name: Windows CI
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test:
+    runs-on: windows-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+      -
+        name: Choco install dependencies
+        uses: crazy-max/ghaction-chocolatey@v2
+        with:
+          args: install openssl cmake
```

Comment:
openssl version?

Reply:
Per chocolatey https://community.chocolatey.org/packages/openssl this will install OpenSSL 1.1.

I've never written any Powershell but I will try to write a script to download it and eventually get us off choco